crates/anvil-polkadot/src/substrate_node/service/mod.rs

```diff
@@ -48,11 +69,228 @@ pub struct Service {
     pub genesis_block_number: u64,
 }
 
+async fn resolve_fork_hash_http(
```

Comment:
I would move all this new methods related to state fetching from an RPC node in a parachain we want to fork from in a different file. Better to start a directory called `fork` with a mod.rs, and we'll submodules, one being `fetch` maybe?, with all these new fns.